code/web/src/modules/user/EditProfile.js

```diff
@@ -53,84 +54,90 @@ class EditProfile extends Component {
 
   // //do we need onChangeSelect? Seems only diff is adding a parseInt so may not need
 
-  // onSubmit = (event) => {
-  //   event.preventDefault()
-    
-  //   this.setState({
-  //     isLoading: true,
-  //   })
+  onSubmit = (event) => {
+    event.preventDefault()
+
+    let newProfileData = this.state.newProfileData
```

Comment:
I think we will be able to thin this on submit function love that we are reusing most of the projects functionality